docs/gigabyte-rgb-fusion2-guide.md

```diff
@@ -29,8 +29,10 @@ Gigabyte RGB Fusion 2.0 5702 Controller
 
 ## Lighting
 
-The controllers have built-in support for six color modes: `off`, `fixed`, `pulse`, `flash`, `double-flash` and `color-cycle`. The `color-cycle` mode fades between all
-color hues, but the extra `fusion_rgb_cycle` script can be used to fade between specific colors.
+The controllers have built-in support for six color modes: `off`, `fixed`, `pulse`,
+`flash`, `double-flash` and `color-cycle`. The `color-cycle` mode fades between all
+color hues, but the extra [fusion_rgb_cycle.py](extra/contrib/fusion_rgb_cycle.py)
```

Comment:
I think the relative address needs to start with `../extra`, since we're in `docs/`, not the root.